chaoslib/litmus/http-chaos/helper/http-helper.go

```diff
@@ -167,7 +167,7 @@ func startProxy(experimentDetails *experimentTypes.ExperimentDetails, pid int) e
 	chaosCommand := fmt.Sprintf("%s && sleep 10 && %s && %s", startProxyServerCommand, createProxyCommand, createToxicCommand)
 
 	cmd := exec.Command("/bin/bash", "-c", chaosCommand)
-	log.Infof("[Chaos]: Starting proxy server: %s", cmd.String())
+	log.Infof("[Chaos]: Starting proxy server")
 
 	_, err := cmd.CombinedOutput()
```

Comment:
Can we not use `CombinedOutput()` & instead use `Run()`, that will return standard output & error separately.
Then we can return actual error instead of `exit status 1`, below is a suggestion, please update accordingly & test.
```suggestion
	var stdout, stderr bytes.Buffer
	cmd.Stdout = &stdout
	cmd.Stderr = &stderr
	err = cmd.Run()
	outStr, errStr := stdout.String(), stderr.String()
        if err != nil {
          // if we get standard error then, return the same
          if errStr != "" {
            return "", fmt.Errorf(errStr)
          }
        // if not standard error found, return error
        return "", err
        }
```

Reply:
If you want to take it in another PR, that is also fine, I see it is getting used like this in many places.

Reply:
Added in this PR.